gt/main.c

```diff
@@ -557,7 +724,8 @@ gt_proc(void *arg)
 			}
 
 			if (pkt_info.priority <= 1) {
-				ret = decap_and_fill_eth(m, gt_conf, &pkt_info);
+				ret = decap_and_fill_eth(
+					m, gt_conf, &pkt_info);
```

Comment:
Since there's now no change on this line, I think it can go back to being on one line.